src/PanoImageRenderer/WebGLUtils.js

```diff
@@ -35,6 +36,12 @@ export default class WebGLUtils {
 		gl.attachShader(program, vertexShader);
 		gl.attachShader(program, fragmentShader);
 		gl.linkProgram(program);
+
+		gl.detachShader(program, vertexShader);
+		gl.detachShader(program, fragmentShader);
```

Comment:
@WoodNeck 

Why remove shaders?

Reply:
The thing is that by calling `deleteShader` doesn't mean that it is perfectly removed on the GPU.
As we never use `vertexShader` / `fragmentShader` after we link the program, they're just holding memory without doing nothing.
So deleting them can free more GPU memory.